pkgs/development/python-modules/pydeck/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipykernel
+, ipywidgets
+, jinja2
+, numpy
+, pandas
+, pytestCheckHook
+, pytest
+, traitlets
+}:
+
+buildPythonPackage rec {
+    pname = "pydeck";
+    version = "0.7.1";
+
```

Only [Python 3.7](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/setup.py#L196) and up is supported.

```suggestion
    version = "0.7.1";

    disabled = pythonOlder "3.7";
```